### DISCUSSION TITLE
Group 20 Discussion

### DISCUSSION THREAD

**Discussion by jedijuli** [Digital-Preservation-Coalition/PythonStudyGroups#12] [General]
Session #1 Notes:

Some common reasons that were noted for learning python in this group:
- Independence from IT
- Maintenance of current scripts
- Working with various systems that use python on the back end or APIs (Archives Space, Preservica)
- Incorporating python into workflow
- Developing some basic scripts
- Working with Open source tools that use Python
- digital side to finding aids, better extract metadata, interoperability with systems, mass editing & scripting

Roles per session: note-taker & facilitator
Communication: we will use email and Github discussions for now and investigate Slack as an option for ongoing discussions between sessions. Preference is for keeping things simple and to not have too many places to check communication. Since GitHub is going to be used for most of the study group content, Discussions might be a useful tool to have ongoing code-related chats.

Next session TODOs:
Before the next session members will work through installing Python and provide an update in Discussions with their environment and overall Python set up. We will also start the first exercise developed for the course on GitHub, Deleting Unwanted Files, and will work through it in the next session. 
Other items to discuss at our next meeting: Python emulators, Installation troubleshooting, additional Python courses to take during our training. 

Also there is a Intro to GitHub training page for those new to GitHub or looking for a refresh (like me!): https://github.com/skills/introduction-to-github 

One last TODO for the group is to sign up for meeting roles - I think Alex volunteered for the note-taker for the next session. And we will need a facilitator. I will also create a discussion post to sign up for the future sessions. Thanks!

**Comment by digitensions** [comment]
Hi group 20! Just wanted to say hi! I'll be about this month if you need any python assistance. Will do my best to help! Good luck, Joanna.

**Comment by arhonemus** [comment]
Hello all!

Who is planning on attending the meeting 4/24?

**Comment by dlove-ucalgary** [comment]
Hi everyone, my apologies for not being able to attend the previous two meetings, other work commitments and priorities are sabotaging my involvement. 

I'm going to have to withdraw from the group but I would just like to add that I'm still very interested in all the discussions. Having a continued GitHub discussion and code repository would be awesome as an ongoing resource. For me this works much better as a place I can 'check-in' when available.

Good luck to everyone!
